open-banking-accelerator/accelerators/ob-apim/repository/resources/apis/DynamicClientRegistration/dcr-swagger.yaml

```diff
@@ -0,0 +1,370 @@
+swagger: '2.0'
+info:
+  title: CDR-DynamicClientRegistration
+  description:
+    This specification defines the APIs for Data Holders exposing Dynamic Client Registration endpoints
+  version: '0.1'
+basePath: /open-banking/{version}
+schemes:
+  - https
+consumes:
+  - application/jwt
+produces:
+  - application/json
+paths:
+  /register:
+    post:
+      summary: Register a client using a CDR Register issued Software Statement Assertion
```

Comment:
Shall we check and remove UK and CDS words from swaggers

Reply:
fixed with https://github.com/wso2/financial-open-banking/pull/1/commits/e070a5da1ffe1fe284282bcf48a213cdeedb7547 and https://github.com/wso2/financial-open-banking/pull/1/commits/50e936355424514807e7f93221d739e8c99ec6b4